plugin/src/types.ts

```diff
@@ -10,6 +10,7 @@ export interface ReanimatedPluginOptions {
   relativeSourceLocation?: boolean;
   disableInlineStylesWarning?: boolean;
   processNestedWorklets?: boolean;
+  isWeb?: boolean;
```

Comment:
I think the flag should represent the action (like `processNestedWorklets`), not the target platform. Imagine we introduce a second optimization for web; how should we name it so that the user can enable only one of them? On the other hand, init data (in particular, worklet code as string) is necessary only on native (for now) and probably will never be needed on web.

Reply:
We could do something like `useWebOptimizations` (`shouldBeUseWebOptimizations` 😭). But I think it might be a bit over the top?

Reply:
How about `omitNativeOnlyData`?